app/internal/auth/oauth/oauth.go

```diff
@@ -69,7 +70,11 @@ func (a Authorizer) HandleProviderCallback(ctx *context.C) (*goth.User, error) {
 		return nil, fmt.Errorf("can't data get from session: %s", err)
 	}
 
-	defer a.Cleanup(ctx.W, ctx.R)
+	defer func() {
+		if cerr := a.Cleanup(ctx.W, ctx.R); cerr != nil {
+			analytics.Log(ctx.Ctx).Warnf("Can't cleanup session: %s", cerr)
```

Comment:
invalid operation: ctx (variable of type *invalid type) has no field or method Ctx